src/main.rs

```diff
@@ -23,6 +23,7 @@ mod common;
 mod crypto;
 mod secure_mount;
 mod tpm;
```

Comment:
You should remove the tpm mod as part of this.

Reply:
Ultimately, this should be removed. But if this is removed, for now, the tpm functions in the main function will not work.

Reply:
You have modules `tpm` and `tpm2`.  I don't understand why we wouldn't only have one or the other.

Reply:
@leonjia0112 what are the dependencies from tpm.rs -> main? 

I am not sure why you did not resolve them as part o this patch.